src/Taboo/TabooPlayer.py

```diff
@@ -38,3 +38,7 @@ def ready(self):
     @ready.setter
     def ready(self, flag):
         self.__ready = flag
+
+    @property
```

Comment:
suggestion: remove property?

The rationale might seem arbitrary but I typically use @property as accessors for "self._attr". If we are computing state at run-time, perhaps it's better to make it visible as a function call.

functional-call also makes it apparent that the attribute is not mutable. It's not clear immediately from the users point of view if obj.nrofConns is mutable just by looking at it; "obj.nrofConns()" is certainly not mutable!)

Reply:
I agree.